server/controllers/finance/reports/financial.employee.handlebars

```diff
@@ -43,6 +43,21 @@
         </tr>
       </thead>
       <tbody>
+        {{#if extractEmployee}}
+          <tr>
+            <td colspan="4" class="text-right"><b>{{translate "TABLE.COLUMNS.BALANCE_AT" }} {{date dates.dateFrom }} </b></td>
+            <td class="text-right">
+              {{debcred creditoropeningBalance.debit metadata.enterprise.currency_id}}
```

Comment:
Can you capitalize each word here?  `creditorOpeningBalance` is better than `creditoropeningbalance`.